endpoints/endpoint.go

```diff
@@ -0,0 +1,79 @@
+package endpoint
+
+import (
+	"encoding/json"
+	"math/rand"
+	"net/http"
+	"strings"
+	"time"
+	"fmt"
+)
+
+const shortUrlLength = 4
+
+type StoreHandler struct {
+	Store map[string]string
+}
+
+func NewstoreHandlers() *StoreHandler {
+	return &StoreHandler{
+
+		Store: map[string]string{},
+	}
+
+}
+func (u *StoreHandler) ShortenUrl(w http.ResponseWriter, r *http.Request) {
+	
+	CheckUrl := strings.Split(r.URL.Path, "/")[2]
+
+	if len(CheckUrl) > 1 {
+		var short_url string
```

Comment:
I see you made changes to some other variables, but you missed this one.
```suggestion
		var shortUrl string
```